app/assets/javascripts/modules/dashboard.js

```diff
@@ -130,15 +156,19 @@ angular.module('probedock.dashboard', [ 'probedock.api', 'probedock.orgs', 'prob
         return;
       }
 
+      $scope.chart.options.tooltipTemplate = chartConfig[$scope.chart.params.type].tooltipTemplate;
```

Comment:
It seems a bit dirty to put this code in a function called `fetchMetrics`. I would create another `updateChart` function that is also called in the same circumstances. The first 3 lines of the function could also go in there.